core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/LandingZoneCreationStatusRunner.scala

```diff
@@ -86,6 +86,7 @@ class LandingZoneCreationStatusRunner(
                     CreationStatuses.Error,
                     Some(s"Landing Zone creation failed: $msg")
                   )
+                  .flatMap(_ => billingRepository.updateLandingZoneId(billingProjectName, None))
```

Comment:
Is there any chance that the landing zone still exists in a partial state and we'll be orphaning cloud resources by removing its ID from the billing project record?

Reply:
I do somewhat worry about this. It might be good to run this by @sergiygetlin to get his opinion.

Reply:
I was talking this over with @aherbst-broad yesterday.
Our conclusion at the time was basically that we have to trust the landing zone service to clean up after itself.
If it does, the id is useless anyway, and all that happens with it is that you get a 403 when you try to use it, like when we go to delete it (which is indistinguishable from a 403 for other reasons).